python/ray/_raylet.pyx

```diff
@@ -1173,6 +1173,10 @@ cdef class CoreWorker:
         if self.async_event_loop is None:
             self.async_event_loop = asyncio.new_event_loop()
             asyncio.set_event_loop(self.async_event_loop)
+            # Initialize the async plasma connection in worker
+            # Delayed import due to everything depends on raylet
```

Comment:
I don't understand this comment out of context. What is "everything" and does it depend on the raylet process or some other raylet-related import?